mypy/nullcontext.py

```diff
@@ -0,0 +1,10 @@
+import sys
+from contextlib import contextmanager
+from typing import Iterator
+
+if sys.version_info < (3, 7):
+    @contextmanager
+    def nullcontext() -> Iterator[None]:
+        yield
+else:
+    from contextlib import nullcontext as nullcontext, contextmanager  # noqa: F401
```

Comment:
nit: don't think you need to import contextmanager again. I think you could also move this to `mypy/util.py` instead of making a separate file.

Reply:
Thanks, copy-pasta, will fix.

Re `mypy/util.py`, I followed the pattern put in place by `mypy/ordered_dict.py`. Maybe we want to put both of those in `mypy/backports.py` or `mypy/compat.py`?